docs/source/kinds_of_types.rst

```diff
@@ -236,18 +237,16 @@ Python functions often accept values of two or more different
 types. You can use :ref:`overloading <function-overloading>` to
 represent this, but union types are often more convenient.
 
-Use the ``Union[T1, ..., Tn]`` type constructor to construct a union
-type. For example, if an argument has type ``Union[int, str]``, both
+Use the ``T1 | ... | Tn`` type constructor to construct a union
```

Comment:
```suggestion
Use the ``T1 | ... | Tn`` operator to construct a union
```
Feels odd to call this a "type constructor"

Reply:
Using "operator" here also didn't feel quite right, so I just left out the "type constructor" bit.